### ISSUE TITLE
Is it possible to change article color through .css file in the dark reader mode?

### ISSUE THREAD

**Issue by BEST8OY** [xiaoyifang/goldendict-ng#2052]
Hello
As stated in the title
Is it possible?
The CSS files only apply when the dark reader mode is off and only changing the background color alone through the CSS and not adjusting other colors doesn't make it look any good!
In some sense I want to be able to change these lines defaults
https://github.com/xiaoyifang/goldendict-ng/blob/861c1d434d1075bf986aca3a59b9b0122c75e2be/src/scripts/darkreader.js#L328-L338

**Comment by BEST8OY** [comment]
I checked the .JS file it seems these parameters are also valid for custom .JS file

https://github.com/xiaoyifang/goldendict-ng/blob/861c1d434d1075bf986aca3a59b9b0122c75e2be/src/scripts/darkreader.js#L4726C1-L4736C7

But why isn't it working?
I also tried using the snippet in this page
https://xiaoyifang.github.io/goldendict-ng/topic_userstyle/
and just to test if those parameters alone work, I changed sepia to 100 but it still didn't apply to GD!
Both latest from git and latest stable release!
![image](https://github.com/user-attachments/assets/fb89ccbf-7266-4141-84d5-27a803c3f774)

**Comment by BEST8OY** [comment]
Microsoft Copilot is really handy :)
I changed this
https://github.com/xiaoyifang/goldendict-ng/blob/861c1d434d1075bf986aca3a59b9b0122c75e2be/src/scripts/darkreader.js#L339-L364
To this
```
const DEFAULT_THEME = {
    mode: 1,
    brightness: 100,
    contrast: 100,
    grayscale: 0,
    sepia: 0,
    useFont: false,
    fontFamily: isMacOS
        ? "Helvetica Neue"
        : isWindows
            ? "Segoe UI"
            : "Open Sans",
    textStroke: 0,
    engine: ThemeEngine.dynamicTheme,
    stylesheet: "",
    darkSchemeBackgroundColor: "#181a1b", // default value
    darkSchemeTextColor: "#e8e6e3", // default value
    lightSchemeBackgroundColor: DEFAULT_COLORS.lightScheme.background,
    lightSchemeTextColor: DEFAULT_COLORS.lightScheme.text,
    scrollbarColor: isMacOS ? "" : "auto",
    selectionColor: "auto",
    styleSystemControls: !isCSSColorSchemePropSupported,
    lightColorScheme: "Default",
    darkColorScheme: "Default",
    immediateModify: false
};

function enable(themeOptions = {}, fixes = null) {
    const theme = {
        ...DEFAULT_THEME,
        ...themeOptions,
        darkSchemeBackgroundColor: themeOptions.darkSchemeBackgroundColor || DEFAULT_THEME.darkSchemeBackgroundColor,
        darkSchemeTextColor: themeOptions.darkSchemeTextColor || DEFAULT_THEME.darkSchemeTextColor
    };
    if (theme.engine !== ThemeEngine.dynamicTheme) {
        throw new Error("Theme engine is not supported.");
    }
    createOrUpdateDynamicThemeInternal(theme, fixes, isIFrame);
    isDarkReaderEnabled = true;
}
```
Built it and now it does what I want!
But the custom js file in style folder not applying is still there!

![image](https://github.com/user-attachments/assets/0143e5e0-d4ca-4035-b97c-347c4b41758a)

**Comment by xiaoyifang** [comment]
![image](https://github.com/user-attachments/assets/ce8f17c7-8fbb-42f5-812c-5a1772a39f29)

article-script.js should not placed under style folder.

**Comment by BEST8OY** [comment]
> article-script.js should not placed under style folder. ![image](https://private-user-

Yeah, sorry for the mistake.
If the feature to change those parameter I mentioned is not gonna get implemented, please close the issue.
Thanks

**Comment by xiaoyifang** [comment]
> If the feature to change those parameter I mentioned is not gonna get implemented, please close the issue.

you can submit a PR,and illustrate what your changes doing.

**Comment by shenlebantongying** [comment]
The code in https://github.com/xiaoyifang/goldendict-ng/pull/2065 already exists somewhere else.
https://github.com/xiaoyifang/goldendict-ng/blob/0543cb2dc8349f2d78c4ce746876409101722e69/src/scripts/darkreader.js#L7750-L7757

Just put the js file in the correct folder as mentioned in the doc.

**Comment by BEST8OY** [comment]
> Just put the js file in the correct folder as mentioned in the doc.

Yep, it all stemmed from me not putting it in the correct place! 
It would be nice to mention what parameters are customizable in the docs!
Not everyone can actually search the code!
